src/abstractdataframe/abstractdataframe.jl

```diff
@@ -784,8 +784,11 @@ end
 """
     vcat(dfs::AbstractDataFrame...)
 
-Vertically concatenate `AbstractDataFrames` that have the same column names in
-the same order.
+Vertically concatenate `AbstractDataFrames`.
+
+Column names in all passed `DataFrames` must be the same, but they can have
```

Comment:
Better say "data frame" since they are not necessarily of the `DataFrame` type. "case" -> "cases" (I think)

Reply:
fixed